targets/XPlatCppSdk/source/code/include/playfab/QoS/XPlatSocket.h

```diff
@@ -66,6 +66,9 @@ namespace PlayFab
 #if defined(PLAYFAB_PLATFORM_WINDOWS) || defined(PLAYFAB_PLATFORM_XBOX)
             int s, slen;
             WSADATA wsa;
+#elif defined(PLAYFAB_PLATFORM_ANDROID)
+            unsigned int s;
+            socklen_t slen;
```

Comment:
Did we check that QoS actually works on Android ? Or is this just to get past compile ?

Reply:
Yes, the test case of QoS passed.

---
In reply to: [273691254](https://github.com/PlayFab/SDKGenerator/pull/470#discussion_r273691254) [](ancestors = 273691254)